dll/cpl/desk/lang/ru-RU.rc

```diff
@@ -89,12 +89,12 @@ BEGIN
     CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR1_B, 184, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
     LTEXT "Цвет &2", IDC_ADVAPPEARANCE_COLOR2_T, 216, 160, 40, 9
     CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR2_B, 216, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
-    LTEXT "Шрифт", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9//FIXME: add accel
+    LTEXT "Шрифт", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9//FIXME: add accel. I used "&Шрифт" for the backport
     COMBOBOX IDC_ADVAPPEARANCE_FONT_C, 5, 200, 130, 90, CBS_DROPDOWN |
              CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Размер", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9//FIXME: add accel
+    LTEXT "Размер", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9//FIXME: add accel. I used "Р&азмер" for the backport. Attention: &P is in use already.
     COMBOBOX IDC_ADVAPPEARANCE_FONTSIZE_E, 142, 200, 38, 200, CBS_DROPDOWN | WS_VSCROLL
-    LTEXT "Цвет", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9//FIXME: add accel
+    LTEXT "Цвет", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9//FIXME: add accel. I used "&Цвет" for the backport
```

Comment:
```suggestion
    LTEXT "Цв&ет", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9//FIXME: I used "&Цвет" for the backport
```

Reply:
Thank you, I switched over to your suggestion based on russian XPSP3. Resolved.